__init__.py

```diff
@@ -79,12 +80,18 @@ class MetroidPrimeWorld(World):
     }
     starting_room_data: Optional[StartRoomData] = None
     prefilled_item_map: Dict[str, str] = {}  # Dict of location name to item name
+    elevator_mapping: Dict[str, Dict[str, str]] = default_elevator_mappings
 
     def get_filler_item_name(self) -> str:
         return SuitUpgrade.Missile_Expansion.value
 
     def generate_early(self) -> None:
         init_starting_room_data(self)
+        info(f"{self.multiworld.get_player_name(self.player)}'s Metroid Prime starting room data: {self.starting_room_data.name}")

```

Comment:
If we get reports of a certain seed not generating, at least we'll have this info in the log to help debug